spec/build-visible-spec.js

```diff
@@ -60,10 +60,7 @@ describe('Visible', function() {
 
     describe('when build panel is toggled and it is visible', function() {
       beforeEach(function () {
-        atom.config.set('build.panelVisibility', 'Keep Visible');
```

Comment:
In the specs the "keep visible" option is used as a way to make the panel visible. Instead, toggle-panel can be triggered.